brick/brick.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main(){
+	var a,b,p,q,r float64
+	fmt.Println("Enter the hole dimensions (2 numbers): ")
+	fmt.Scan(&a,&b)
+	fmt.Println("Enter the brick dimensions (3 numbers): ")
+	fmt.Scan(&p,&q,&r)
+	if a >= p || a >= q || a >= r || b >= p || b >= q || b >= r {
```

Comment:
This is not quite correct. Check the following example:

* Hole dimensions: 2, 3
* Brick dimensions: 3, 4, 5

Your program outputs `The brick can be inserted in the hole!`, which is false.